dockerfiles/hpcc/templates/eclagent.yaml

```diff
@@ -14,6 +14,7 @@ spec:
         run: {{ .name | quote }}
     spec:
       serviceAccountName: hpcc
+      {{- include "hpcc.checkDataStorageHostMount" (dict "root" $) | indent 6 }}
```

Comment:
We don't normally use a dict when only passing one value - but perhaps doing so when the param is something other than . is not a bad convention

Reply:
not sure, makes the template use of .root. a little clearer I think (and common with other places that use top)